app/containers/DashboardContainer/index.js

```diff
@@ -10,21 +10,35 @@ import * as ToastActions from '../../actions/toasts';
 
 import Dashboard from '../../components/Dashboard';
 
-const DashboardContainer = ({
-  actions,
-  dashboard,
-  settings,
-  history,
-  musicSources
-}) => (
-  <Dashboard
-    dashboardData={dashboard}
-    history={history}
-    settings={settings}
-    actions={actions}
-    musicSources={musicSources}
-  />
-);
+class DashboardContainer extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    this.props.actions.readFavorites();
```

Comment:
If we want to know if some tracks are among favorites, we need to fetch them in DashboardContainer.